run-tests.sh

```diff
@@ -33,6 +33,15 @@ check_manifest () {
     check-manifest
 }
 
+check_dockerfile () {
+    docker run -i --rm hadolint/hadolint < Dockerfile
+}
+
+check_dockerbuild () {
```

Comment:
Very minor detail: in `reana` this line is named `check_docker_build`. To make things consistent, which name is better and to which one we should stick to?

Reply:
It can go either way: 
- I had preferred `--check-dockerbuild` over `--check-docker-build` in one word so that it would be harmonious with `--check-dockerfile`. Basically, all the commands are being named `--check-something` where "something" is always one word, not sometimes one word sometimes two words.  
- But there is also a good point (=readability) behind going for `--check-docker-build`, `--check-js-tests`, etc...

Reply:
Since you have more PRs already done, I can simply amend towards `docker-build` with hyphen in this one :wink: